# HG changeset patch
# User Rik <rik@octave.org>
# Date 1380774561 25200
#      Wed Oct 02 21:29:21 2013 -0700
# Node ID b29c8a067c116e73b79aefd037e2d687d44df435
# Parent  4f57d4de0383adb7c9b55c3cf9a67b1252e7e3f7
legend.m: Remove leftover code bits from debugging.

* scripts/plot/legend.m: Remove leftover code bits from debugging.

diff --git a/scripts/plot/legend.m b/scripts/plot/legend.m
--- a/scripts/plot/legend.m
+++ b/scripts/plot/legend.m
@@ -386,19 +386,18 @@ function [hleg, hleg_obj, hplot, labels]
             have_dname = true;
             break;
           endif
         elseif (strcmp (typ, "hggroup"))
           hgkids = get (hkid, "children");
           for j = 1 : length (hgkids)
             try
               dname = get (hgkids(j), "DisplayName");
-              if (! isempty (dname));
+              if (! isempty (dname))
                 have_dname = true;
-                toc
                 break;  # break from j-loop over hgkids
               endif
             end_try_catch
           endfor
           if (have_dname)
             break;  # break from k loop over nkids
           endif
         endif  # elseif hggroup
